.github/workflows/R-CMD-check.yaml

```diff
@@ -31,6 +31,8 @@ jobs:
           - {os: macOS-latest,   pandoc: '2.7.3',   r: 'release'}
           - {os: ubuntu-18.04,   pandoc: '2.7.3',   r: 'oldrel',  rspm: "https://packagemanager.rstudio.com/cran/__linux__/bionic/latest"}
           - {os: ubuntu-18.04,   pandoc: '2.7.3',   r: 'devel',   rspm: "https://packagemanager.rstudio.com/cran/__linux__/bionic/latest", http-user-agent: "R/4.0.0 (ubuntu-18.04) R (4.0.0 x86_64-pc-linux-gnu x86_64 linux-gnu) on GitHub Actions" }
+          - {os: ubuntu-18.04,   pandoc: '2.0',   r: 'release', rspm: "https://packagemanager.rstudio.com/cran/__linux__/bionic/latest"}
```

Comment:
But this won't work as you explained ? File is not in the good format for the action to download. 
https://github.com/jgm/pandoc/releases/tag/2.0

Also I don't want us to test all Pandoc version. We are running CI regularly and it becomes a lot of running times (and resources). 

Why is our aim with that ? 

We could also add a Weekly or Monthly run dedicated to Pandoc version on only one version of R. This would reduce the load for each commit or PR.

Reply:
I feel it is reasonable to test minimal 2.x version.
At the same time, I agree it is better to avoid testing too many versions for every commits and PRs.
The addition of 2.0 is the temporary trial, and I will remove it.
I am okay to remove 2.0.0.1 as well.

Reply:
yeah we can add one. I'll open a new issue to rediscuss which version we want to test. 

I did 2.7.3 mainly because it was the default in the RStudio IDE but it has now upgraded.

Reply:
great. thanks for your neat comments.